drivers/javascript/util.coffee

```diff
@@ -122,6 +123,23 @@ mkSeq = (response, opts) -> recursivelyConvertPseudotype(response.r, opts)
 mkErr = (ErrClass, response, root) ->
     new ErrClass(mkAtom(response), root, response.b)
 
+module.exports.asCallback = (promise, callback) ->
+  if typeof callback !== 'function' return promise
+  promise.then((value) -> callback(null, value), callback)
```

Comment:
Should we consider `.catch` here and include error handling? e.g. https://github.com/petkaantonov/bluebird/blob/3746b7eca90dd8b11af73db5d30cf46d7dd90f9b/src/nodeify.js#L39

Reply:
curious how errors aren't being handled here?